authors/apps/articles/serializers.py

```diff
@@ -18,14 +19,16 @@ class CreateArticleSerializer(serializers.ModelSerializer):
     class Meta:
         model = Article
 
-        fields = ['title', 'description', 'body', 'author_id']
+        fields = ['title', 'description', 'body', 'author_id', ]
 
     def validate(self, data):
         author_id = data.pop('author_id', None)
         author = User.objects.get(id=author_id)
         data.update({'author': author})
-
         return data
 
     def create(self, validated_data):
+        read_time = readtime.of_text(validated_data.get('body'),wpm = 265)
```

Comment:
@NicholusMuwonge, great work.

Kindly share where you got the wpm of 265.
Also look into whether this is representative by researching on it.
Please do this ASAP so I can merge this and so as to merge @richien's as well.

Reply:
I got 256 wpm from medium support center https://help.medium.com/hc/en-us/articles/214991667-Read-time